### PR TITLE
Bold online users in Roomauth

### PR DESCRIPTION
**Pull request by sparkychild**
This allows people who want to appeal their roomban to be able to identify the users that are online in that room.   I hope this is the most efficient way of doing it.

### CODE REVIEW

**Comment by xfix** [comment]
Pretty sure that won't work, considering `Users.users` is now a map.

I'm also not sure if Showdown needs a feature to display which staff members are online. I for instance don't want to be bothered by random PMs from some random guy who realized that I was online, despite not being in any of public rooms.

**Comment by sparkychild** [comment]
Would using ``u in targetRoom.users`` be better as it checks the rooms userlist rather than the all the users on the server?   Also sorry about the Users.users thing; I forgot it changed.

File: commands.js [hunk 1]
```diff
@@ -790,7 +790,7 @@ exports.commands = {
 		let rankLists = {};
 		for (let u in targetRoom.auth) {
 			if (!rankLists[targetRoom.auth[u]]) rankLists[targetRoom.auth[u]] = [];
-			rankLists[targetRoom.auth[u]].push(u);
+			rankLists[targetRoom.auth[u]].push((u in targetRoom.users ? "**" + u + "**" : u));
```
Comment (kotarou3) [inline commands.js]: You don't need the extra set of brackets

**Comment by kotarou3** [comment]
Always test against the latest `master` branch of the server. (e.g., `git rebase` to it)

Also, since this is a small minor commit, keep it as a single commit by squashing it

**Comment by Zarel** [comment]
@sparkychild, we're still waiting for you to squash and fix the issues @kotarou3 commented about.

File: commands.js (hunk 1 above)
Comment (Zarel) [inline commands.js]: The extra brackets don't aid readability at all. On the other hand, if you wanted, you could put

`.push((u in targetRoom.users) ? "**" + u + "**" : u)`

But that's not really necessary, either.

**Comment by sparkychild** [comment]
I'll get this done as quickly as i can. Having trouble trying to squash/rebase on a mac

**Comment by panpawn** [comment]
I helped squash heh

(Praise Cloud9 /o/)

**Comment by TheImmortal** [comment]
This breaks the alphabetical sorting.

**Pull request merged**